node18/README.md

```diff
@@ -0,0 +1,59 @@
+## Node.js toolbox
+
+### Info
+This is an image of Node.js 16.x version  in toolbox format
+
+### Global npm packages
+This image has pre-installed npm global packages
+
+ - yarn
+ - gulp-cli
+ - grunt-cli
+ - node-sass
+ - bower
+ - typescript
+ - tslint
+
+### Configuration
+Available binary paths for export:
+
+- /usr/bin/node
+- /usr/bin/npm
+- /usr/bin/yarn
+- /usr/bin/gulp
+- /usr/bin/grunt
+- /usr/bin/node-sass
+- /usr/bin/bower
+- /usr/bin/uglifycss
+- /usr/bin/uglifyjs
+- /usr/bin/tsc
+- /usr/bin/tslint
+- /usr/bin/npx
+
+### Sample configuration
+```
+version: '2.4'
+services:
+  node:
+    image: nfqlt/node16
+    volumes:
+      - './src:/home/project/src'
+      - /tmp
+
+  dev:
+    image: nfqlt/php73-dev
```

Comment:
Older images are not updated, but for non-php development, we have nfqlt/dev image, example should reflect that.

Reply:
Updated